app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java

```diff
@@ -57,6 +72,61 @@ protected void onCreate(Bundle savedInstanceState) {
         saveButton = findViewById(R.id.memory_save_button);
 
 
+        View.OnClickListener cameraClick = new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                 lastClick= (ImageButton) v;
+                new AlertDialog.Builder(MemoryEditActivity.this)
+                        .setTitle("Add Media")
```

Comment:
You should actually use string resources for this, so that we can easily add multiple languages.

In strings.xml:
```<string name="dialog_add_memory_media">Add Media</string>```

Then replace the line above (`.setTitle("Add Media")`) with `.setTitle(R.string.dialog_add_memory_media)`

Same should be done for the `setMessage`, `setPositiveButton` and `setNegativeButton`.